shex/invoke.js

```diff
@@ -5,13 +5,13 @@
 var component = require("../components/shex-cmumps-to-rdf");
 var fs = require('fs');
 
-var data = fs.readFileSync(process.argv[2]);
+var data = fs.readFileSync(process.argv[2], "utf8");
 
 var ret = component.updater(data);
 if (ret instanceof Promise) {
 
   ret.then( function (r) { 
-      console.warn("resolved:", r.length, "chars") 
+      console.warn("resolved:", r, "chars") 
```

Comment:
Looks like you are dumping the full object, but still have chars on it.  Maybe remove that ,"chars" from the end of the console.warn line?